plugins/drupal-plugin/README.md

```diff
@@ -123,7 +123,7 @@ All the modules should now be installed and activated. To ensure that they are a
 
    <img src="./docs/images/setup-step-9.png" width="500">
 
-1. Head to the bottom on the page and click the ****Install**** button.
+1. Head to the bottom on the page and click the \***\*Install\*\*** button.
```

Comment:
I'm not sure if this was intended to be `***` but it looks like prettier is trying to correct this syntax

Reply:
Yeah pretty sure it was just meant to be bolded